### PR TITLE
fix p2p selection statement in getWaveformSNR()

### PR DESCRIPTION
**Pull request by MyoungchulK**
By following this line : `if(TMath::Abs(y-yVals[firstBin]>p2p)){ `in `getWaveformSNR()`,
That line is checking whether the new p2p value is bigger than the previous p2p value or not.
But it looks like it is comparing the two values by the inside of `TMath::Abs()`
In this case, the new p2p value can be still in a negative state during the comparison.
Basically, it is ignoring all 'negative slope or downgoing p2p(+ to -)' results(if it is not intended).
In order to compare them in the right way, I think it should be `if ( TMath::Abs(y - yVals[firstBin]) > p2p ){`.

### CODE REVIEW

**Comment by cozzyd** [comment]
yes the parentheses are certainly in the wrong place currently.. not sure anybody really uses this method and would be affected it by it suddenly starting to work correctly?

**Comment by MyoungchulK** [comment]
Let's say 
`y = -800`, 
`yVals[firstBin] =800`
and `p2p = 1500`

If we follow original line, it will become `TMath::Abs(-800 - 800 > 1500)` -> `TMath::Abs(-1600 > 1500)`
Event though new p2p is `1600`, but it is still having minus sign(`-1600`). 
Then, that 'if statement' will still see new p2p is smaller than the previous p2p  and not move on to the line inside of the 'if statement'
`p2p=TMath::Abs(y-yVals[firstBin])`;
So, basically, it is not recording 1600 as a new p2p. 

I think it will work correctly by that small changes (ppl are not using that much though:)). 
Based on my python copy, it is showing the correct value.
